ui/raidboss/data/04-sb/raid/o12s.js

```diff
@@ -66,10 +66,10 @@
     },
     {
       id: 'O12S Solar Ray',
-      regex: / 14:(?:3350|3351):(?:Omega|Omega-M) starts using (?:Unknown_3350|Unknown_3351|Solar Ray) on (\y{Name})/,
-      regexDe: / 14:(?:3350|3351):(?:Omega|Omega-M) starts using (?:Unknown_3350|Unknown_3351|Sonnenstrahl) on (\y{Name})/,
-      regexFr: / 14:(?:3350|3351):(?:Oméga|Oméga-M) starts using (?:Unknown_3350|Unknown_3351|Rayon solaire|Rayon Solaire) on (\y{Name})/,
-      regexJa: / 14:(?:3350|3351):(?:オメガ|オメガM) starts using (?:Unknown_3350|Unknown_3351|ソーラレイ) on (\y{Name})/,
+      regex: / 14:(?:3350|3351):(?:Omega|Omega-M) starts using Solar Ray/,
+      regexDe: / 14:(?:3350|3351):(?:Omega|Omega-M) starts using Sonnenstrahl/,
+      regexFr: / 14:(?:3350|3351):(?:Oméga|Oméga-M) starts using Rayon Solaire/,
+      regexJa: / 14:(?:3350|3351):(?:オメガ|オメガM) starts using ソーラレイ/,
```

Comment:
These lines lost the `on (\y{Name})` capturing group.

Reply:
done :)